private/buf/cmd/buf/command/push/push.go

```diff
@@ -133,6 +134,17 @@ func (f *flags) Bind(flagSet *pflag.FlagSet) {
 		"Do not use. This flag never had any effect",
 	)
 	_ = flagSet.MarkHidden(trackFlagName)
+	flagSet.SetNormalizeFunc(aliasDraftToBranchFunc)
+}
+
+// aliasDraftToBranchFunc aliases the draft flag to branch.
```

Comment:
- are we sure we want to directly alias, and have no change in behavior?
- is there not a native way in pflag/cobra to do this?

Reply:
For the initial step of releasing the branches UI, yes. There are server side changes we need to make for upcoming features, but it doesn't warrant a flag structure change.

This is the recommended way from Cobra's `pflag` package: https://github.com/spf13/pflag#mutating-or-normalizing-flag-names

Reply:
Once we do this, it's forever, so we can't later bring in any changes to `--branch` - just confirming we're comfortable with this

Reply:
I had an original draft of this change where I added a new `--branch` flag, and then had a check return an error if the user attempted to pass both a `--draft` and `--branch`.
The normalize func here seems to be the recommended way to do a direct alias, but I think I would be more comfortable with the new flag and the check. Does that seem reasonable to you?

Reply:
Yea I would error if both are specified.